https://bugzilla.mozilla.org/show_bug.cgi?id=436806

objformat has been deprecated for a while. elf has been the default format for
FreeBSD for a while too. objformat is gone from FreeBSD 7, so we default to elf.

--- nss-3.12/mozilla/security/coreconf/FreeBSD.mk.old	2008-09-05 11:38:11 +0200
+++ nss-3.12/mozilla/security/coreconf/FreeBSD.mk	2008-09-05 11:38:28 +0200
@@ -65,7 +65,7 @@
 
 ARCH			= freebsd
 
-MOZ_OBJFORMAT		:= $(shell test -x /usr/bin/objformat && /usr/bin/objformat || echo aout)
+MOZ_OBJFORMAT		:= $(shell test -x /usr/bin/objformat && /usr/bin/objformat || echo elf)
 
 ifeq ($(MOZ_OBJFORMAT),elf)
 DLL_SUFFIX		= so
